config/kcp/apibinding.yaml

```diff
@@ -5,9 +5,11 @@ metadata:
 spec:
   reference:
     workspace:
-      path: WORKSPACE
       exportName: data.my.domain
-  acceptedPermissionClaims:
+  permissionClaims:
```

Comment:
In my opinion we should start tagging versions of the example with the release match.
go.mod is still with github.com/kcp-dev/kcp/pkg/apis v0.7.0, which has `acceptedPermissionClaims`
There is PR #30  that addresses the update of the versions. IMHO PR #30 could be merged first, which would accommodate the change here.

Reply:
I'm +1 to making sure the releases match up to KCP's